tensorflow_addons/image/color_ops.py

```diff
@@ -31,6 +31,7 @@ def _scale_channel(
     image: TensorLike, channel: int, bins: Optional[int] = 256
 ) -> tf.Tensor:
     """Scale the data in the channel to implement equalize."""
+    assert bins >= 256, "Number of histogram bins should be at least 256"
```

Comment:
I suppose this is not the case for one of the API that expose bins params like scikit-image that we could already use in tests here https://scikit-image.org/docs/dev/api/skimage.exposure.html#skimage.exposure.equalize_hist

Reply:
assertion has been removed